content/root/platform-customization.textile

```diff
@@ -0,0 +1,147 @@
+- *Regional routing of traffic*: Require that all traffic is routed and serviced within a specific region, such as the EU or US.
+- *Regional message storage*: Require that all messages are stored within a specific region.
+
+A custom environment provides you with your own endpoints to service Ably platform requests. These can be either subdomains of @ably.io@, or point at your own custom CNAME domain. If you choose the latter option, we can also customize our client library SDKs to use your chosen domain and make them available via our CDN as @cdn.ably.io/lib/yourcompany.min.js@.
+
+If you're ready to enable a custom environment to take advantage of these features, then please "see our guide":#custom-environment-setup. Otherwise, read on to find out more.
+
+
+h2(#active-traffic-management). Active traffic management
+
+Active traffic management (ATM) is a feature available exclusively to our Enterprise customers that enables the Ably engineering team to automatically and manually manage the endpoints that customers connect to.
+
+This ensures that under adverse conditions, such as a denial of service attack or a major outage, Ably can proactively route Enterprise customer traffic to specific data centers or regions that are unaffected, or even to new temporary clusters. This enables customers to maintain service while the issue is investigated and resolved.
+
+This service is ideal for customers who want the piece of mind that Ably actively monitors not only the global cluster health but also the endpoints used by that customer, and can act to isolate or move traffic, ensuring business continuity.
+
+h2(#dedicated-and-isolated-clusters). Dedicated, isolated clusters
+
+As part of our Enterprise package offering, we are able to offer customers dedicated clusters that are either partially or completely isolated from Ably's global cluster.  Customers who require isolation from our global cluster for security, governance or guaranteed capacity reasons are able to run the Ably platform on their own EC2 dedicated environments.  
+
+The levels of isolation Ably can offer, from least to most isolated, are:
+
+- Dedicated realtime and routing servers, but shared database cluster and common services (such as stats, logging, registries, configuration)
+- Dedicated realtime and routing servers, dedicated database cluster and shared common services
+- Dedicated realtime and routing servers, dedicated database and common services
+ 
+Ably can run a dedicated cluster on behalf of our Enterprose customers, or if required, directly on instances provided to us by our customers using their Amazon EC2 account.
+
+We provide you with custom endpoints for your cluster, for example: @yourcompany-realtime.ably.io@ and @yourcompany-realtime.ably.io@.  Additionally, we can support CNAMEs using your own custom root domain.
+
+Please note the following:
+- Ably only supports Amazon EC2 environments (within Virtual Private Networks)
```

Comment:
Do you mean VPC (Virtual Private Cloud) here? You can access EC2 via a VPN but can create an EC2 instance within a VPC

Reply:
@markcoxsmith This info was gleaned from [this FAQ](https://faqs.ably.com/do-you-support-dedicated-or-isolated-clusters) which states VPNs .. can you confirm that this should read VPC?

Reply:
Doesn't seem to be massively important either in the terminology and I was being overly nit-picky. Some of the other info mentioned on that FAQ is suspect though as per the following convo: [https://ably-real-time.slack.com/archives/CNHDHKRFB/p1637253165438900](https://ably-real-time.slack.com/archives/CNHDHKRFB/p1637253165438900)

Reply:
Thanks @markcoxsmith: I have addressed these points [here](https://github.com/ably/docs/pull/1253/commits/a95c758a5a58aba2a5aca922e8f325ea43efa515)